extra-resources/mods/Mythos/common.cardset

```diff
@@ -118,9 +145,38 @@ card('Healer') {
         }
     }
 }
-
+// https://upload.wikimedia.org/wikipedia/commons/e/ec/Ricci_Guangqi_2.jpg
+// CC BY-SA 3.0
+// https://commons.wikimedia.org/wiki/User:SteveMcCluskey
```

Comment:
Crap. In all cases of CC photos I linked the "Uploaded By" which seemed the most natural, not knowing there was a "Use the file" link. What do you suggest?